nginx-serve/Dockerfile

```diff
@@ -0,0 +1,57 @@
+FROM node:18-bullseye AS runtime
+
+RUN apt-get update -y \
+    && apt-get install -y --no-install-recommends \
+        git bash g++ make \
+    && rm -rf /var/lib/apt/lists/*
+
+WORKDIR /code
+
+RUN git config --global --add safe.directory /code
+
+# ------------------------------------------------------------------------------------
+FROM runtime AS nginx-build
+
+# TODO: Add cache with packages.json
+# This is not working, will need @samshara help to configure this
+# COPY package.json yarn.lock /code/
+# COPY app/package.json /code/app/
```

Comment:
This could be because, we have multiple package.json files. It's also inside packages/ui